libc/src/__support/str_to_float.h

```diff
@@ -1055,6 +1056,37 @@ hexadecimal_string_to_float(const char *__restrict src,
   return output;
 }
 
+struct NanMantissaResult {
+  uint64_t value;
+  int error;
+
+  LIBC_INLINE constexpr NanMantissaResult(uint64_t value, int error)
+      : value(value), error(0) {}
+};
+
+LIBC_INLINE NanMantissaResult
+nan_mantissa_from_ncharseq(const cpp::string_view ncharseq) {
```

Comment:
`const` qualifier for value type in the function signature is not needed / effective.

Reply:
done